pyracing/response_objects/upcoming_events.py

```diff
@@ -109,14 +111,17 @@ def __init__(self, dict):
                 self.standing_start = dict['standingStart']
                 self.temp_unit = dict['weatherTempUnits']
                 self.temp_value = dict['weatherTempValue']
-                self.time_end = dict['endTime']
+                self.time_end = datetime_from_iracing_timestamp(
+                    dict['endTime'])
                 self.time_of_day = dict['timeOfDay']
-                self.time_start = dict['startTime']
-                self.time_start_sim = dict['simulatedstarttime']
-                self.track = dict['trackName']
-                self.track_config = dict['trackConfigName']
+                self.time_start = datetime_from_iracing_timestamp(
+                    dict['startTime'])
+                self.time_start_sim = parse_encode(dict['simulatedstarttime'])
```

Comment:
should this be a `datetime_from_iracing_timestamp` call instead of `parse_encode`?

Reply:
You can see the example value here: https://esterni.github.io/pyracing/response-objects/race_guide/#race

Classic iRacing fashion, it's inconsistent with other implementations